### PR TITLE
changed generator to create JSON test light curves

### PR DESCRIPTION
**Pull request by kstoreyf**

### CODE REVIEW

File: generate_PSC.py [hunk 1]
```diff
@@ -39,20 +40,29 @@
     obstime = obstime + gap
     metadata['TIME{:04d}'.format(t)] = obstime.isoformat()
 hdu.header.extend(metadata)
+print(metadata)
 
-hdu.writeto('PSC_0001.fits', clobber=True)
+hdu.writeto('PSC_0002.fits', clobber=True)
```
Comment (joshwalawender) [inline generate_PSC.py]: A to do item should be to figure out a naming convention for PSC files.

File: generate_PSC.py [hunk 2]
```diff
@@ -39,20 +40,29 @@
...
         sig_b = 0.017
-        r = np.random.normal(1,sig_r)
-        g = np.random.normal(1,sig_g)
-        b = np.random.normal(1,sig_b)
-        FO.write('{:26s} {:6.3f} {:6.3f} {:6.3f} {:6.3f} {:6.3f} {:6.3f}\n'.format(
-                 time, r, g, b, sig_r, sig_g, sig_b))
+        r = np.random.normal(1, sig_r)
+        g = np.random.normal(1, sig_g)
+        b = np.random.normal(1, sig_b)
+        entry = {
```
Comment (joshwalawender) [inline generate_PSC.py]: We should figure out how to format and add metadata (clone of the fits file header?) to this JSON entry.

**Comment by wtgee** [comment]
@joshwalawender Let's add that as an Issue.

Also, are we adding the two Green channels or do we want two separate green values? I assume adding.

**Comment by kstoreyf** [comment]
@wtgee @joshwalawender i added this as an issue - i'm going to go ahead and create issues for the other things we discussed // that are currently in TODOs in the design doc

**Comment by wtgee** [comment]
@kstoreyf Sounds good.

:+1: for the merge at this point. @joshwalawender ?

File: generate_PSC.py [hunk 3]
```diff
@@ -4,6 +4,7 @@
 from astropy.io import fits
 import astropy.units as u
 from astropy.coordinates import SkyCoord
+import json
 
 ## Generate Fake Postange Stamp Cube (FITS cube)
```
Comment (mimming) [inline generate_PSC.py]: Consider adding some usage instructions

File: generate_PSC.py (hunk 1 above)
Comment (mimming) [inline generate_PSC.py]: Consider making the output file name a command line argument

File: generate_PSC.py [hunk 4]
```diff
@@ -4,6 +4,7 @@
 from astropy.io import fits
 import astropy.units as u
 from astropy.coordinates import SkyCoord
+import json
```
Comment (mimming) [inline generate_PSC.py]: Are you using a style guide?

This isn't something we've discussed on this project, but we might as well start going with one.  This one seems popular https://www.python.org/dev/peps/pep-0008/

**Pull request merged**